src/glue/jobs/s3_to_json_s3.py

```diff
@@ -363,30 +363,31 @@ def is_expected_validation_error(validation_result, client_info):
     """
     if not validation_result["errors"]:
         return False
-    else:
-        if "Android" not in client_info:
+    if validation_result["appId"] != "mobile-toolbox":
+        return False
+    if "Android" not in client_info:
+        return False
+    if "metadata.json" in validation_result["errors"]:
+        metadata_errors = validation_result["errors"]["metadata.json"]
+        if not (len(metadata_errors) == 2 and
+                "'appName' is a required property" in metadata_errors and
+                "'files' is a required property" in metadata_errors):
+            return False
+    if "taskData.json" in validation_result["errors"]:
```

Comment:
Nit. Use .get(....) then you don't need the extra if statement.

Reply:
Normally, yes. But I'm checking for inequality (`if not errors == ...`) so unless I made the non-severe error the default return value of the `get` method then in the case that there were no taskData.json errors, the inequality would evaluate to true and `is_expected_validation_error` would prematurely return False. The way it's written now is more readable and less redundant imo than duplicating the non-severe errors in the arguments to `get`. Although, I think the `not ==` syntax is a bit unconventional for single comparisons, so I will change that.